test/string_view-test/string_view-test.cpp

```diff
@@ -0,0 +1,343 @@
+/*
+* string_view-test.cpp
+* Sean Murthy, Ray Chen
+* (c) 2020 sigcpp https://sigcpp.github.io. See LICENSE.MD
+*
+* Attribution and copyright notice must be retained.
+* - Attribution may be augmented to include additional authors
+* - Copyright notice cannot be altered
+* Attribution and copyright info may be relocated but they must be conspicuous.
+*
+* Test string_view template
+* - see C++17 [string.view]
+*	https://timsong-cpp.github.io/cppwp/n4659/string.view
+*/
+
+#include <stdexcept>
+
+#include "../verifiers.h"
+
+void string_view_test()
+{
+	using std::string_view;
+
+	//1. Construction and assignment
+	//2. Capacity
+
+	// Default constructor. Constructs an empty string_view. 
+	string_view empty_sv;
+	is_zero(empty_sv.size(), "empty_sv.size()");
+	is_true(empty_sv.length() == empty_sv.size(), "empty_sv.length() = empty_sv.size()");
+	is_true(empty_sv.empty(), "empty_sv.empty()");
+
+	//constructs a string_view with const char
```

Comment:
`"hello world"` is a `const char` array

```suggestion
	//constructs a string_view with const char[]
```